DEV_GUIDE.md

```diff
@@ -110,16 +126,19 @@ therefore they do not need any extra quoting.
 ```sh script
 env JAVA_OPTS='-Dmy_property="some value"' ./starctl --cluster-name 'Some Cluster' ...
```

Comment:
So you will maintain `starctl` and it will start the coordinator, yes?

Reply:
yes. I guess I should make script changes in this PR as well